docs/cog_guides/dev.rst

```diff
@@ -0,0 +1,251 @@
+does not persist through restarts/shutdowns.
+
+**Arguments**
+
+* ``[toggle]``: |bool-input| Otherwise, defaults to the inverse of the current setting.
+
+.. _dev-command-debug:
+
+^^^^^
+debug
+^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]debug <code>
+
+**Description**
+
+Evaluate a statement of python code.
+
+The bot will always respond with the return value of the code.
+If the return value of the code is a coroutine, it will be awaited,
+and the result of that will be the bot's response.
+
+Note: Only one statement may be evaluated. Using certain restricted
+keywords, e.g. yield, will result in a syntax error. For multiple
+lines or asynchronous code, see [p]repl or [p]eval.
+
+**Environment Variables**
+
+* ``ctx``: Command invocation context
+* ``bot``: The bot object
+* ``channel``: The current channel object
+* ``author``: The current author's member object
+* ``guild``: The current guild object
+* ``message``: The command's message object
+* ``aiohttp``: The aiohttp library
+* ``asyncio``: The asyncio library
+* ``discord``: The discord.py library
+* ``commands``: The redbot.core.commands module
+* ``cf``: The redbot.core.utils.chat_formatting module
+* ``_``: The result from the last dev command
+
+**Arguments**
+
+* ``<code>``: The code to debug.
+
+.. _dev-command-eval:
+
+^^^^
+eval
+^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]eval <body>
+
+**Description**
+
+Execute asynchronous code.
+
+This command wraps code into the body of an async function and then
+calls and awaits it. The bot will respond with anything printed to
+stdout, as well as the return value of the function.
+
+The code can be within a codeblock, inline code or neither, as long
+as they are not mixed and they are formatted correctly.
+
+**Environment Variables**
+
+* ``ctx``: Command invocation context
+* ``bot``: The bot object
+* ``channel``: The current channel object
+* ``author``: The current author's member object
+* ``guild``: The current guild object
+* ``message``: The command's message object
+* ``aiohttp``: The aiohttp library
+* ``asyncio``: The asyncio library
+* ``discord``: The discord.py library
+* ``commands``: The redbot.core.commands module
+* ``cf``: The redbot.core.utils.chat_formatting module
+* ``_``: The result from the last dev command
+
+**Arguments**
+
+* ``<body>``: The code to evaluate.
+
+.. _dev-command-mock:
+
+^^^^
+mock
+^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]mock <user> <command>
+
+**Description**
+
+Mock another user invoking a command. The prefix must not be entered.
+
+**Arguments**
+
+* ``<member>``: The user to mock. |user-input-quotes|
+* ``<command>``: The command to invoke.
+
+.. _dev-command-mockmsg:
+
+^^^^^^^
+mockmsg
+^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]mockmsg <user> <content>
+
+**Description**
+
+Dispatch a message event as if it were sent by a different user.
+
+Only reads the raw content of the message. Attachments, embeds etc. are
```

Comment:
Incorrect?

Reply:
Seems this docstring was updated some time ago and I hadn't updated it here, apologies, will fix now